src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -711,35 +711,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         Private Shared ReadOnly s_conflictMarkerLength As Integer = "<<<<<<<".Length
 
         Private Function IsConflictMarkerTrivia() As Boolean
-            If CanGet() Then
-                Dim ch = Peek()
-
-                If ch = "<"c OrElse ch = ">"c OrElse ch = "="c Then
-                    Dim position = _lineBufferOffset
-                    Dim text = _buffer
-
-                    If position = 0 OrElse SyntaxFacts.IsNewLine(text(position - 1)) Then
-                        Dim firstCh = _buffer(position)
-
-                        If (position + s_conflictMarkerLength) <= text.Length Then
-                            For i = 0 To s_conflictMarkerLength - 1
-                                If text(position + i) <> firstCh Then
-                                    Return False
-                                End If
-                            Next
-
-                            If firstCh = "="c Then
-                                Return True
-                            End If
-
-                            Return (position + s_conflictMarkerLength) < text.Length AndAlso
-                                   text(position + s_conflictMarkerLength) = " "c
-                        End If
-                    End If
-                End If
+            Dim ch as Char = Nothing
+            If Not IsAtNewLine OrElse Not TryGet(0, ch) OrElse (ch <> "<"c AndAlso ch <> ">"c AndAlso ch <> "="c) Then
+                Return False
             End If
-
-            Return False
+            If Not (NextAre("<<<<<<<") OrElse NextAre(">>>>>>>") OrElse NextAre("=======")) Then
+                Return False
+            End If
+            If ch = "="c Then
+                Return True
+            End If
+            Return NextIs(s_conflictMarkerLength, " "c)
```

Comment:
this brings this method out of sync with the C# version.  i would prefer they stay in sync so that if we need to change one, we can easily change both.

Reply:
I've synced the implementations